### DISCUSSION TITLE
Time series parameters for AutoGluonTabular

### DISCUSSION THREAD

**Discussion by fsty** [autogluon/autogluon#2968] [Q&A] [answered]
tabular_hyperparmeters hase no effect, all default models will be trained  
```
predictor = TimeSeriesPredictor(
    path="ts/", 
    prediction_length=12,
    target="QTE",
    eval_metric="RMSE", 
    known_covariates_names=["act"],
    quantile_levels=[0.1, 0.5, 0.9], 
    validation_splitter=splitter, 
    verbosity=4
).fit(
    train_data, 
     hyperparameters={
      "AutoGluonTabular": {"tabular_hyperparmeters": {"XGB":{}}  },
      }, 
      time_limit=1200
)
```

**Comment by shchur** [comment]
Thanks for pointing this out! I've opened an issue #2969 to track this problem.

**Comment by shchur** [comment, marked as answer]
There is a typo in your code, should be `tabular_hyperparameters` instead of `tabular_hyperparmeters`.

**Reply by fsty** [reply]
Sorry about this, it was a copy/paste from https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-model-zoo.html#autogluon.timeseries.models.AutoGluonTabularModel

**Reply by shchur** [reply]
Oh, sorry, thanks for pointing this out! Will try to submit a fix soon.
